### PR TITLE
Add new publisher Vogue

### PR DESCRIPTION
**Pull request by Oreo7985**

### CODE REVIEW

File: src/fundus/publishers/de/vogue.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+class VogueParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = XPath("//article//p")
+        _subheadline_selector = XPath("//article//h2")
```
Comment (MaxDall) [inline src/fundus/publishers/de/vogue.py:15]: This newspaper seems to support a little `summary` as well. An example would be [this article](https://www.vogue.de/artikel/jennifer-garner-outfit-hellblau-rot-sommer-2024). In Fundus context the summary would be
```
Hellblau und Rot wirken als Trendfarben allein schon stilvoll – Jennifer Garner beweist nun, wie genial die Kombination der beiden im Frühjahr/Sommer 2024 sein kann.
```

Further the selectors  are a bit too unspecific, extracting a lot of text unrelated to the article like

```
Von Katharina Fuchs
Mehr Themen auf Vogue.de:
Von Katharina Fuchs
Von Katharina Fuchs
Von Gabi Thorne
Von Marie Périer
Von Héloïse Salessy, Vogue Germany
Von Alessandra Signorelli
```

All of those are taken from the article I mentioned above.

**Review by MaxDall — CHANGES_REQUESTED** [review]
Thanks for adding 👍 this one looks really good!

File: src/fundus/publishers/de/__init__.py [hunk 2]
```diff
@@ -240,3 +241,14 @@ class DE(PublisherEnum):
         ],
         parser=RheinischePostParser,
     )
+
+    Vogue = PublisherSpec(
```
Comment (MaxDall) [inline src/fundus/publishers/de/__init__.py:245]: Could you rename the publisher to `VogueDE`

**Review by MaxDall — CHANGES_REQUESTED** [review]

**Comment by Oreo7985** [comment]
Hi Max, I have fixed the problems you mentioned. Please review the code again and see if the PR can be committed. Thanks! @MaxDall

**Comment by MaxDall** [comment]
@Oreo7985 I adjusted the selectors.

**Review by MaxDall — APPROVED** [review]

**Pull request merged**